tests/e2e-openshift/must-gather/chainsaw-test.yaml

```diff
@@ -0,0 +1,105 @@
+apiVersion: chainsaw.kyverno.io/v1alpha1
+kind: Test
+metadata:
+  name: must-gather
+spec:
+  namespace: chainsaw-must-gather
+  steps:
+  - name: Install Target Allocator
+    try:
+    - apply:
+        template: true
+        file: install-target-allocator.yaml
+    - assert:
+        file: install-target-allocator-assert.yaml
```

Comment:
If this is an assertion file, can we name it `assert-target-allocator.yaml` instead? It's easier to see at a glance what each yaml file represents if it's in the prefix.